modular_skyrat/modules/synths/code/surgery/robot_chassis_restoration.dm

```diff
@@ -1,69 +1,84 @@
 /datum/surgery/positronic_restoration
 	name = "Posibrain Reboot (Revival)"
 	steps = list(
-	/datum/surgery_step/mechanic_unwrench,
-	/datum/surgery_step/pry_off_plating/fullbody,
-	/datum/surgery_step/cut_wires/fullbody,
-	/datum/surgery_step/replace_wires/fullbody,
-	/datum/surgery_step/prepare_electronics,
-	/datum/surgery_step/add_plating/fullbody,
-	/datum/surgery_step/weld_plating/fullbody,
-	/datum/surgery_step/finalize_positronic_restoration,
-	/datum/surgery_step/mechanic_close)
+		/datum/surgery_step/mechanic_unwrench,
+		/datum/surgery_step/pry_off_plating/fullbody,
+		/datum/surgery_step/cut_wires/fullbody,
+		/datum/surgery_step/replace_wires/fullbody,
+		/datum/surgery_step/prepare_electronics,
+		/datum/surgery_step/add_plating/fullbody,
+		/datum/surgery_step/weld_plating/fullbody,
+		/datum/surgery_step/finalize_positronic_restoration,
+		/datum/surgery_step/mechanic_close,
+	)
 
 	target_mobtypes = list(/mob/living/carbon/human)
 	possible_locs = list(BODY_ZONE_CHEST)
 	requires_bodypart_type = BODYTYPE_ROBOTIC
 	desc = "A surgical procedure that reboots a positronic brain."
 
 /datum/surgery/robot_chassis_restoration/can_start(mob/user, mob/living/carbon/target)
-	if(!..())
-		return FALSE
-	if(target.stat != DEAD)
-		return FALSE
-	if(target.suiciding)
-		return FALSE
-	var/obj/item/organ/internal/brain/target_brain = target.getorganslot(ORGAN_SLOT_BRAIN)
-	if(!target_brain)
+	if(!..() || target.stat != DEAD || target.suiciding || !target.getorganslot(ORGAN_SLOT_BRAIN))
 		return FALSE
+
 	return TRUE
```

Comment:
My brain tells me that this should probably be changed to be all on one line, albeit I don't know if it's really helping readability much. So I'm fine with it being this way.